server/src/test/java/org/elasticsearch/cluster/coordination/PreVoteCollectorTests.java

```diff
@@ -112,7 +120,7 @@ public String toString() {
             assert electionOccurred == false;
             electionOccurred = true;
         }, l -> {
-        }, ElectionStrategy.DEFAULT_INSTANCE);
+        }, ElectionStrategy.DEFAULT_INSTANCE, () -> healthStatus); // TODO need tests that check that the max term seen is updated
```

Comment:
Suspect this comment was removed from `master` recently.

```suggestion
        }, ElectionStrategy.DEFAULT_INSTANCE, () -> healthStatus);
```